File: 은지/LV1/3진법 뒤집기.js
function solution(n) {
  // 10진법 -> 3진법
    const t = n.toString(3)

  // 10진법 변환
  var answer = 0
  for (let i=0; i<t.length; i++) {
    answer += Number(t[i]) * (3 ** i)
  }
  return answer;
}

File: 준영/프로그래머스/LV1/3진법 뒤집기.py
def solution(n):
    answer = 0
    arr = [0] * 19

    idx = 0
    while n:
        arr[idx] = n // (3 ** (18-idx))
        n %= (3 ** (18-idx))
        idx += 1
        
    for i in range(19):
        if arr[i] != 0:
            new_arr = arr[i:]
            break

    for i in range(len(new_arr)):
        answer += ((3 ** i) * new_arr[i])

    
    return answer

print(solution(45))
